### PR TITLE
setup gitlab CI

### PR DESCRIPTION
**Pull request by rasolca**

### CODE REVIEW

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try-

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

File: ci/docker/codecov/deploy.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+RUN mkdir ${BUILD} && cd ${BUILD} && \
+    CC=${MPICH_PATH}/bin/mpicc CXX=${MPICH_PATH}/bin/mpicxx FC=${MPICH_PATH}/bin/mpif90 cmake ${SOURCE} \
+      -DCMAKE_BUILD_TYPE=Debug \
+      -DCMAKE_CXX_FLAGS="-O0 -Werror -fprofile-arcs -ftest-coverage" \
+      -DCMAKE_EXE_LINKER_FLAGS="-fprofile-arcs -ftest-coverage" \
+      -DDLAI_CI_RUNNER_USES_MPIRUN=ON \
+      -DDLAI_WITH_MKL=0 \
+      -DLAPACK_CUSTOM_TYPE=Custom \
+      -DLAPACK_CUSTOM_INCLUDE_DIR=${OPENBLAS_PATH}/include \
+      -DLAPACK_CUSTOM_LIBRARY="-L${OPENBLAS_PATH}/lib;openblas" \
```
Comment (albestro) [inline ci/docker/codecov/deploy.Dockerfile:34]: In the last commits of the CMake refactoring I changed this.

```suggestion
      -DLAPACK_LIBRARY="-L${OPENBLAS_PATH}/lib;openblas" \
```

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try

**Comment by rasolca** [comment]
bors try-

**Comment by rasolca** [comment]
bors try

**Review by albestro — APPROVED** [review]
not dig into all details, but it LGTM

**Pull request merged**